Apps/W1/EDocument/app/src/Processing/Import/PrepareDraft/EDocProviders.Codeunit.al

```diff
@@ -171,4 +184,25 @@ codeunit 6124 "E-Doc. Providers" implements IPurchaseLineProvider, IUnitOfMeasur
                 exit(true);
     end;
 
+    procedure GetDefaultUnitOfMeasure(PurchaseLineType: Enum "Purchase Line Type"; PurchaseLineTypeNo: Code[20]) UnitOfMeasureCode: Code[20];
+    var
+        Item: Record Item;
+        Resource: Record Resource;
+    begin
+        case PurchaseLineType of
```

Comment:
Can we move this up to when we find an item reference without UOM? 
That would mean move this code up somewhere at line 184 and change that a bit.